liferay-tango/liferay-portal
# LPS-122300 [BUG] [Content Dashboard] Fix searching problems in Filter by Subtype.

**Motivation**

The filters inside the modal "filter by subtype" in Content Dashboard is not working properly.

**How to test it**

1. Create some Web Content subtype: subtype1
2. Go to Content Dashboard and filter by subtype
3. Check that in the list appears our subtype subtype1
4. Search subtype in the up part of the modal
5. Expected behavior
6. The searched subtype appears in the modal
7. If the user clicks in clear, cancels the search but the modal remains opened
8. When there is no result the message "There is no subtype" appears
